Handle stale issues/PRs

johnnyaug opened the pull request:
Closes #6900.

N-o-Z reviewed (changes requested):
Very cool!

Can we add a workflow dispatch to it also so we can test it?

johnnyaug commented:
@N-o-Z I intend to add it on a side branch after the initial merge - don't want the dispatch on master

nopcoder reviewed (approved):
Cool

arielshaqed commented on .github/workflows/close-stale-issues.yaml:
```diff
@@ -0,0 +1,23 @@
+        - name: Close stale issues and PRs
+          uses: actions/stale@v8
+          with:
+          stale-issue-message: 'Marking this issue as stale after 30 days of inactivity, please comment or this will be closed.'
```
Here are some issues that document known deficiencies of lakeFS.  Of course we could close them every 30 days and open a new one, or add comments every 29 days.  But *why* do I need to work for the tooling?

Some of these have user requirements or discussions on them, it would be particularly painful to lose those.

* #1371 
* #1405
* #1605
* #1607
* #1726
* #1742 
* #1765
* #1804
* #1849 
* #1911 
* #1971 
* #2092 
* #2150
* #2228 
* #2308 
* #2491 
*

arielshaqed reviewed (changes requested):
I see disadvantages to closing issues automatically.  They lose information and cost us community good-will.  I am not sure that the advantages outweigh them; I think we should list the advantages of closing issues like this!

johnnyaug commented:
@arielshaqed @N-o-Z for now, changed this action to only affect pull requests (not issues). I think we can reach a consensus about that. 
Added an item to the upcoming R&D meeting to discuss applying to issues as well.

arielshaqed reviewed (approved):
Thanks!

N-o-Z reviewed (approved):
Thanks!
